src/bcs/MFEMScalarFunctorBoundaryIntegratedBC.C

```diff
@@ -0,0 +1,40 @@
+#include "MFEMScalarFunctorBoundaryIntegratedBC.h"
+
+registerMooseObject("PlatypusApp", MFEMScalarFunctorBoundaryIntegratedBC);
+
+InputParameters
+MFEMScalarFunctorBoundaryIntegratedBC::validParams()
+{
+  InputParameters params = MFEMIntegratedBC::validParams();
+  params.addClassDescription("Adds the domain integrator to an MFEM problem for the linear form "
+                             "$(f, v)_\\Omega$ "
+                             "arising from the weak form of the forcing term $f$.");
+  params.addRequiredParam<MFEMScalarCoefficientName>(
+      "coefficient",
+      "The functor which will be used in the integrated BC. A functor is any of the following: a "
+      "variable, an MFEM material property, a function, or a post-processor.");
```

Comment:
As above - something like:
```suggestion
      "The coefficient which will be used in the integrated BC. A coefficient is can be any of the following: a "
      "variable, an MFEM material property, a function, or a post-processor.");
```

Reply:
I've addressed this in a commit of my own.